### ISSUE TITLE
Code fails to start

### ISSUE THREAD

**Issue by BenjaminLeseur** [microsoft/vscode#76679]
Issue Type: <b>Bug</b>

Bug in my previous Code profile's files, this morning, code do not start.
I use Ubuntu 18.04. To start it again, I renamed $HOME/.config/Code to Code_old.

Log :
[2019-07-05 10:39:58.184] [main] [debug] Starting VS Code
[2019-07-05 10:39:58.184] [main] [debug] from: /usr/share/code/resources/app
[2019-07-05 10:39:58.184] [main] [debug] args: {"_":[],"diff":false,"add":false,"goto":false,"new-window":false,"reuse-window":false,"wait":false,"version":false,"help":false,"list-extensions":false,"show-versions":false,"verbose":true,"status":false,"prof-startup":false,"disable-extensions":false,"disable-gpu":false,"logExtensionHostCommunication":false,"skip-getting-started":false,"skip-release-notes":false,"sticky-quickopen":false,"disable-restore-windows":false,"disable-telemetry":false,"disable-updates":false,"disable-crash-reporter":false,"skip-add-to-recently-opened":false,"unity-launch":false,"open-url":false,"nolazy":false,"issue":false,"file-write":false,"file-chmod":false,"driver-verbose":false,"force":false,"prof-code-loading":false}
[2019-07-05 10:39:58.184] [main] [trace] Resolving machine identifier...
[2019-07-05 10:39:58.184] [main] [trace] Resolved machine identifier: 70b76834eee6ea50c2d1971f1bda56e8ab48cfeb19f7d7af098a0f8b94cc214e
[2019-07-05 10:39:58.189] [main] [info] update#setState idle
[2019-07-05 10:39:58.190] [main] [trace] [storage state.vscdb] open(/home/idobs4/.config/Code/User/globalStorage/state.vscdb, retryOnBusy: true)
[2019-07-05 10:39:58.190] [main] [trace] lifecycle (main): phase changed (value: 2)
[2019-07-05 10:39:58.190] [main] [trace] windowsManager#open
[2019-07-05 10:39:58.256] [main] [trace] telemetry/optInStatus {"optIn":true}
[2019-07-05 10:39:58.256] [main] [trace] telemetry/machineIdFallback {"usingFallbackGuid":false}
[2019-07-05 10:39:58.257] [main] [error] URIError: URI malformed
    at decodeURIComponent (<anonymous>)
    at s (/usr/share/code/resources/app/out/vs/code/electron-main/main.js:106:693)
    at Function.parse (/usr/share/code/resources/app/out/vs/code/electron-main/main.js:111:310)
    at Object.t.restoreRecentlyOpened (/usr/share/code/resources/app/out/vs/code/electron-main/main.js:226:857)
    at e.getRecentlyOpenedFromStorage (/usr/share/code/resources/app/out/vs/code/electron-main/main.js:385:457)
    at e.addEntriesFromStorage (/usr/share/code/resources/app/out/vs/code/electron-main/main.js:385:101)
    at e.addRecentlyOpened (/usr/share/code/resources/app/out/vs/code/electron-main/main.js:383:115)
    at t.open (/usr/share/code/resources/app/out/vs/code/electron-main/main.js:448:116)
    at t.openFirstWindow (/usr/share/code/resources/app/out/vs/code/electron-main/main.js:490:304)
    at a.invokeFunction.t (/usr/share/code/resources/app/out/vs/code/electron-main/main.js:484:931)
    at l.invokeFunction (/usr/share/code/resources/app/out/vs/code/electron-main/main.js:228:522)
    at t.<anonymous> (/usr/share/code/resources/app/out/vs/code/electron-main/main.js:484:908)
    at Generator.next (<anonymous>)
    at o (/usr/share/code/resources/app/out/vs/code/electron-main/main.js:13:454)
[2019-07-05 10:39:58.257] [main] [trace] Lifecycle#kill()

VS Code version: Code 1.36.0 (0f3794b38477eea13fb47fbe15a42798e6129338, 2019-07-03T13:20:56.241Z)
OS version: Linux x64 4.15.0-54-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz (4 x 3524)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: disabled_software<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: unavailable_off<br>viz_display_compositor: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 0|
|Memory (System)|23.45GB (18.49GB free)|
|Process Argv|--verbose|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (35)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|2.0.5
vscode-css-formatter|aes|1.0.1
vscode-eclipse-keybindings|alp|0.10.0
vscode-intelephense-client|bme|1.1.1
log-wrapper-for-vscode|chr|1.0.2
jshint|dba|0.10.20
vscode-eslint|dba|1.9.0
codeception-snippets|die|0.0.4
xml|Dot|2.5.0
vscode-html-css|ecm|0.2.0
prettier-vscode|esb|1.9.0
php-debug|fel|1.13.0
php-intellisense|fel|2.3.10
php-pack|fel|1.0.2
auto-rename-tag|for|0.1.0
todo-tree|Gru|0.0.138
beautify|Hoo|1.5.0
volt|hun|0.0.1
phpcs|ika|1.0.5
smarty|imp|0.3.0
php-cs-fixer|jun|0.1.119
favorites|kdc|2.4.5
vscode-phpfmt|kok|1.0.30
PHPDocument|man|1.0.3
PHPDocument2|man|1.0.1
php-import-checker|mar|0.3.0
php-namespace-resolver|Meh|1.1.8
dotenv|mik|1.0.1
php-docblocker|nei|2.0.0
text-power-tools|qcz|1.11.1
format-html-in-php|rif|1.5.3
vscode-nginx|sha|0.6.0
vscode-icons|vsc|8.8.0
gitblame|wad|2.8.1
zephir|zep|1.6.2

</details>

**Comment by JStiller** [comment]
I got also an Problem by starting VSCode. I use Windows 10.

``` [main 2019-07-05T11:41:43.096Z] Starting VS Code
[main 2019-07-05T11:41:43.097Z] from: c:\dev\Microsoft VS Code\resources\app
[main 2019-07-05T11:41:43.097Z] args: { _: [],
  diff: false,
  add: false,
  goto: false,
  'new-window': false,
  'reuse-window': false,
  wait: false,
  version: false,
  help: false,
  'list-extensions': false,
  'show-versions': false,
  verbose: true,
  status: false,
  'prof-startup': false,
  'disable-extensions': false,
  'disable-gpu': false,
  logExtensionHostCommunication: false,
  'skip-getting-started': false,
  'skip-release-notes': false,
  'sticky-quickopen': false,
  'disable-restore-windows': false,
  'disable-telemetry': false,
  'disable-updates': false,
  'disable-crash-reporter': false,
  'skip-add-to-recently-opened': false,
  'unity-launch': false,
  'open-url': false,
  nolazy: false,
  issue: false,
  'file-write': false,
  'file-chmod': false,
  'driver-verbose': false,
  force: false,
  'prof-code-loading': false }
[main 2019-07-05T11:41:43.101Z] Resolving machine identifier...
[main 2019-07-05T11:41:43.102Z] Resolved machine identifier: bc183c89f3a61f735558810cc0cc6c67e677e03f03dd48f8b1ef5196c6096f36
[main 2019-07-05T11:41:43.125Z] update#setState idle
[main 2019-07-05T11:41:43.126Z] [storage state.vscdb] open(c:\Users\JStiller\AppData\Roaming\Code\User\globalStorage\state.vscdb, retryOnBusy: true)
[main 2019-07-05T11:41:43.127Z] lifecycle (main): phase changed (value: 2)
[main 2019-07-05T11:41:43.129Z] windowsManager#open
[main 2019-07-05T11:41:43.211Z] telemetry/optInStatus { optIn: true }
[main 2019-07-05T11:41:43.225Z] telemetry/update:win32SetupTarget { target: 'user' }
[main 2019-07-05T11:41:43.225Z] telemetry/machineIdFallback { usingFallbackGuid: false }
[main 2019-07-05T11:41:43.228Z] URIError: URI malformed
    at decodeURIComponent (<anonymous>)
    at s (C:\dev\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:106:693)
    at Function.parse (C:\dev\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:111:310)
    at Object.t.restoreRecentlyOpened (C:\dev\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:226:857)
    at e.getRecentlyOpenedFromStorage (C:\dev\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:385:457)
    at e.addEntriesFromStorage (C:\dev\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:385:101)
    at e.addRecentlyOpened (C:\dev\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:383:115)
    at t.open (C:\dev\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:448:116)
    at t.openFirstWindow (C:\dev\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:490:304)
    at a.invokeFunction.t (C:\dev\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:484:931)
    at l.invokeFunction (C:\dev\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:228:522)
    at t.<anonymous> (C:\dev\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:484:908)
    at Generator.next (<anonymous>)
    at o (C:\dev\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:13:454)
[main 2019-07-05T11:41:43.229Z] Lifecycle#kill()
[main 2019-07-05T11:41:43.260Z] Lifecycle#window.on('closed') - window ID 1
[main 2019-07-05T11:41:43.260Z] Lifecycle#onWillShutdown.fire()
[main 2019-07-05T11:41:43.276Z] [storage state.vscdb] close()
```

**Comment by arjunagbt** [comment]
Same problem, using macOS Mojave 10.14.4

```
[950:0707/180842.654688:WARNING:dns_config_service_posix.cc(181)] dns_config has unhandled options!
[main 2019-07-07T11:08:42.769Z] Starting VS Code
[main 2019-07-07T11:08:42.771Z] from: /Applications/Visual Studio Code.app/Contents/Resources/app
[main 2019-07-07T11:08:42.771Z] args: { _: [],
  diff: false,
  add: false,
  goto: false,
  'new-window': false,
  'reuse-window': false,
  wait: false,
  version: false,
  help: false,
  'list-extensions': false,
  'show-versions': false,
  verbose: true,
  status: false,
  'prof-startup': false,
  'disable-extensions': false,
  'disable-gpu': false,
  logExtensionHostCommunication: false,
  'skip-getting-started': false,
  'skip-release-notes': false,
  'sticky-quickopen': false,
  'disable-restore-windows': false,
  'disable-telemetry': false,
  'disable-updates': false,
  'disable-crash-reporter': false,
  'skip-add-to-recently-opened': false,
  'unity-launch': false,
  'open-url': false,
  nolazy: false,
  issue: false,
  'file-write': false,
  'file-chmod': false,
  'driver-verbose': false,
  force: false,
  'prof-code-loading': false }
[main 2019-07-07T11:08:42.773Z] Resolving machine identifier...
[main 2019-07-07T11:08:42.773Z] Resolved machine identifier: 95f7c0bef80e1ffa55fdfc48dabfc0b8ed6c0b3b744c56489ecfb301b0c00859
[main 2019-07-07T11:08:42.794Z] update#setState idle
[main 2019-07-07T11:08:42.807Z] [storage state.vscdb] open(/Users/arjuna/Library/Application Support/Code/User/globalStorage/state.vscdb, retryOnBusy: true)
[main 2019-07-07T11:08:42.808Z] lifecycle (main): phase changed (value: 2)
[main 2019-07-07T11:08:42.808Z] windowsManager#open
2019-07-07 18:08:42.816 Electron[950:16161] *** WARNING: Textured window <AtomNSWindow: 0x7fce69c472e0> is getting an implicitly transparent titlebar. This will break when linking against newer SDKs. Use NSWindow's -titlebarAppearsTransparent=YES instead.
[main 2019-07-07T11:08:42.865Z] telemetry/optInStatus { optIn: true }
[main 2019-07-07T11:08:42.866Z] telemetry/machineIdFallback { usingFallbackGuid: false }
[main 2019-07-07T11:08:42.866Z] URIError: URI malformed
    at decodeURIComponent (<anonymous>)
    at s (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:106:693)
    at Function.parse (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:111:310)
    at Object.t.restoreRecentlyOpened (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:226:857)
    at e.getRecentlyOpenedFromStorage (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:385:457)
    at e.addEntriesFromStorage (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:385:101)
    at e.addRecentlyOpened (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:383:115)
    at t.open (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:448:116)
    at t.openFirstWindow (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:490:304)
    at a.invokeFunction.t (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:484:931)
    at l.invokeFunction (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:228:522)
    at t.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:484:908)
    at Generator.next (<anonymous>)
    at o (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:13:454)
[main 2019-07-07T11:08:42.867Z] Lifecycle#kill()
[main 2019-07-07T11:08:42.870Z] Lifecycle#window.on('closed') - window ID 1
[950:0707/180842.900630:ERROR:mach_port_broker.mm(175)] Unknown process 953 is sending Mach IPC messages!
[main 2019-07-07T11:08:42.983Z] [storage state.vscdb] Trace (event): PRAGMA user_version = 1;
```

**Comment by v3ritas1989** [comment]
Windows 10, same problem

```
[main 2019-07-05T14:40:33.741Z] Starting VS Code
[main 2019-07-05T14:40:33.742Z] from: c:\Users\username\AppData\Local\Programs\Microsoft VS Code\resources\app
[main 2019-07-05T14:40:33.742Z] args: { _: [],
  diff: false,
  add: false,
  goto: false,
  'new-window': false,
  'reuse-window': false,
  wait: false,
  version: false,
  help: false,
  'list-extensions': false,
  'show-versions': false,
  verbose: true,
  status: false,
  'prof-startup': false,
  'disable-extensions': false,
  'disable-gpu': false,
  logExtensionHostCommunication: false,
  'skip-getting-started': false,
  'skip-release-notes': false,
  'sticky-quickopen': false,
  'disable-restore-windows': false,
  'disable-telemetry': false,
  'disable-updates': false,
  'disable-crash-reporter': false,
  'skip-add-to-recently-opened': false,
  'unity-launch': false,
  'open-url': false,
  nolazy: false,
  issue: false,
  'file-write': false,
  'file-chmod': false,
  'driver-verbose': false,
  force: false,
  'prof-code-loading': false }
[main 2019-07-05T14:40:33.746Z] Resolving machine identifier...
[main 2019-07-05T14:40:33.746Z] Resolved machine identifier: someidentifierID
[main 2019-07-05T14:40:33.769Z] update#setState idle
[main 2019-07-05T14:40:33.773Z] [storage state.vscdb] open(c:\Users\user\AppData\Roaming\Code\User\globalStorage\state.vscdb, retryOnBusy: true)
[main 2019-07-05T14:40:33.774Z] lifecycle (main): phase changed (value: 2)
[main 2019-07-05T14:40:33.775Z] windowsManager#open
[main 2019-07-05T14:40:33.831Z] telemetry/optInStatus { optIn: true }
[main 2019-07-05T14:40:33.831Z] telemetry/update:win32SetupTarget { target: 'user' }
[main 2019-07-05T14:40:33.831Z] telemetry/machineIdFallback { usingFallbackGuid: false }
[main 2019-07-05T14:40:33.832Z] URIError: URI malformed
    at decodeURIComponent (<anonymous>)
    at s (C:\Users\userr\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:106:693)
    at Function.parse (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:111:310)
    at Object.t.restoreRecentlyOpened (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:226:857)
    at e.getRecentlyOpenedFromStorage (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:385:457)
    at e.addEntriesFromStorage (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:385:101)
    at e.addRecentlyOpened (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:383:115)
    at t.open (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:448:116)
    at t.openFirstWindow (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:490:304)
    at a.invokeFunction.t (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:484:931)
    at l.invokeFunction (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:228:522)
    at t.<anonymous> (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:484:908)
    at Generator.next (<anonymous>)
    at o (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:13:454)
[main 2019-07-05T14:40:33.833Z] Lifecycle#kill()
[main 2019-07-05T14:40:33.841Z] Lifecycle#window.on('closed') - window ID 1
[main 2019-07-05T14:40:33.841Z] Lifecycle#onWillShutdown.fire()
[main 2019-07-05T14:40:33.845Z] [storage state.vscdb] close()
```

**Comment by v3ritas1989** [comment]
```[main 2019-07-05T14:40:33.741Z] Starting VS Code[main 2019-07-05T14:40:33.742Z] from: c:\Users\username\AppData\Local\Programs\Microsoft VS Code\resources\app[main 2019-07-05T14:40:33.742Z] args: { _: [],  diff: false,  add: false,  goto: false,  'new-window': false,  'reuse-window': false,  wait: false,  version: false,  help: false,  'list-extensions': false,  'show-versions': false,  verbose: true,  status: false,  'prof-startup': false,  'disable-extensions': false,  'disable-gpu': false,  logExtensionHostCommunication: false,  'skip-getting-started': false,  'skip-release-notes': false,  'sticky-quickopen': false,  'disable-restore-windows': false,  'disable-telemetry': false,  'disable-updates': false,  'disable-crash-reporter': false,  'skip-add-to-recently-opened': false,  'unity-launch': false,  'open-url': false,  nolazy: false,  issue: false,  'file-write': false,  'file-chmod': false,  'driver-verbose': false,  force: false,  'prof-code-loading': false }[main 2019-07-05T14:40:33.746Z] Resolving machine identifier...[main 2019-07-05T14:40:33.746Z] Resolved machine identifier: someidentifierID[main 2019-07-05T14:40:33.769Z] update#setState idle[main 2019-07-05T14:40:33.773Z] [storage state.vscdb] open(c:\Users\user\AppData\Roaming\Code\User\globalStorage\state.vscdb, retryOnBusy: true)[main 2019-07-05T14:40:33.774Z] lifecycle (main): phase changed (value: 2)[main 2019-07-05T14:40:33.775Z] windowsManager#open[main 2019-07-05T14:40:33.831Z] telemetry/optInStatus { optIn: true }[main 2019-07-05T14:40:33.831Z] telemetry/update:win32SetupTarget { target: 'user' }[main 2019-07-05T14:40:33.831Z] telemetry/machineIdFallback { usingFallbackGuid: false }[main 2019-07-05T14:40:33.832Z] URIError: URI malformed    at decodeURIComponent (<anonymous>)    at s (C:\Users\userr\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:106:693)    at Function.parse (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:111:310)    at Object.t.restoreRecentlyOpened (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:226:857)    at e.getRecentlyOpenedFromStorage (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:385:457)    at e.addEntriesFromStorage (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:385:101)    at e.addRecentlyOpened (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:383:115)    at t.open (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:448:116)    at t.openFirstWindow (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:490:304)    at a.invokeFunction.t (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:484:931)    at l.invokeFunction (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:228:522)    at t.<anonymous> (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:484:908)    at Generator.next (<anonymous>)    at o (C:\Users\user\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\code\electron-main\main.js:13:454)[main 2019-07-05T14:40:33.833Z] Lifecycle#kill()[main 2019-07-05T14:40:33.841Z] Lifecycle#window.on('closed') - window ID 1[main 2019-07-05T14:40:33.841Z] Lifecycle#onWillShutdown.fire()[main 2019-07-05T14:40:33.845Z] [storage state.vscdb] close()```

**Comment by aeschli** [comment]
Can you attach `storage.json` from `$HOME/.config/Code_old/` so I can find out what URI caused that?

@jrieken FYI.

**Comment by MofX** [comment]
It is easy to break even a running instance of code by running eg "code ^%cd^%". This will open a file "%cd%". After exiting it will not start anymore. The entry in the storage.json is something like "file:///c%3A/%cd%25"

**Comment by jrieken** [comment]
Surely triggered by https://github.com/microsoft/vscode/issues/76506

**Comment by arjunagbt** [comment]
This is my storage.json file from `$HOME/Library/Application Support/Code/`
[storage.json](https://gist.github.com/arjunagbt/a7dbd61675d5f8098c3c723d8e637825)

**Comment by jrieken** [comment]
Thanks @arjunagbt. This confirms my theory about #76506 which is about an error that happens when paths contain `%<hex><hex>` sequences. Sorry for the inconvenience, next insiders will have a fix and are working on releasing 1.36.1
